app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerActivityProdTest.kt

```diff
@@ -353,7 +353,7 @@ class NavigationDrawerActivityProdTest {
     }
   }
 
-  // TODO(#1806): Enable this once lowfi implementation is done.
+  // TODO(#552): Enable this once lowfi implementation is done.
```

Comment:
@anandwana001 will these actually be addressed with your changes?

Reply:
No, we need these todo, with my ui driven thing, there will be no my downloads menu.